src/Reflection/ReflectionParameter.php

```diff
@@ -449,6 +449,11 @@ public function canBePassedByValue(): bool
         return ! $this->isPassedByReference();
     }
 
+    public function isPromoted(): bool
+    {
+        return $this->node->flags !== 0;
```

Comment:
Is there a more precise flag value to check? :thinking:

Reply:
Unfortunately not: https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Param.php#L40

Reply:
Yeah, only found one cryptic usage: `$this->semStack[$stackPos-(6-2)]` :D 

Let's roll it as-is :+1: